### PR TITLE
Make app responsive

### PR DESCRIPTION
**Pull request by kapillamba4**
### Description
Make app responsive
### Screenshots
Screen Shot 2020-03-11 at 10 52 39

Screen Shot 2020-03-09 at 23 30 55

### CODE REVIEW

**Comment by byliuyang** [comment]
Hi @kapillamba4 , thank you so much for working on this feature! I have come suggestions on the design:

- Can the button has the same height as the text field for mobile? 
- “New Short Link” title should be centered for mobile phone. 
- In both screenshot, there should be distance between the logo and the edge.

**Comment by kapillamba4** [comment]
> Hi @kapillamba4 , thank you so much for working on this feature! I have come suggestions on the design:
> 
>     * Can the button has the same height as the text field for mobile?
> 
>     * “New Short Link” title should be centered for mobile phone.
> 
>     * In both screenshot, there should be distance between the logo and the edge.

Made changes

File: frontend/src/component/pages/Home.scss [hunk 1]
```diff
@@ -14,6 +14,32 @@
...
       }
     }
+
+    .create-short-link-btn {
+      display: flex;
+      justify-content: center;
+    }
+  }
+
+  @media only screen and (max-width: 480px) {
```
Comment (byliuyang) [inline frontend/src/component/pages/Home.scss]: Nit: I found the following example interesting. Maybe we can try something similar for the next refactor ticket:
https://github.com/sass/sass/issues/554#issuecomment-18306483

File: frontend/src/component/pages/shared/Header.scss [hunk 2]
```diff
@@ -20,6 +20,7 @@ header {
     font-weight: $font-weight-semi-bold;
     letter-spacing: 2px;
     line-height: 30px;
+    margin-left: 8px;
```
Comment (byliuyang) [inline frontend/src/component/pages/shared/Header.scss]: Nit: Let's only add this for mobile devices.

File: frontend/src/component/pages/shared/Header.scss [hunk 3]
```diff
@@ -28,5 +29,6 @@ header {
 
   .sign-out {
     display: flex;
+    margin-right: 8px;
```
Comment (byliuyang) [inline frontend/src/component/pages/shared/Header.scss]: Nit: Let's only add this for mobile devices.

**Comment by byliuyang** [comment]
Ideally `Create Short Link` button should have full width on mobile devices.

**Comment by kapillamba4** [comment]
> Ideally `Create Short Link` button should have full width on mobile devices.

I'll do that

File: frontend/src/component/pages/Home.scss [hunk 4]
```diff
@@ -14,6 +15,41 @@
...
+
+    .control.create-short-link {
+      flex-direction: column;
+      height: 180px;
+
+      .text-field-wrapper {
+        display: flex;
+        margin: 4px;
+
+        input {
```
Comment (byliuyang) [inline frontend/src/component/pages/Home.scss]: I wonder whether we can move this media query into `https://github.com/short-d/short/blob/master/frontend/src/component/form/TextField.scss`

File: frontend/src/component/pages/Home.scss [hunk 5]
```diff
@@ -14,6 +15,41 @@
...
+          width: 100%;
+        }
+      }
+
+      .create-short-link-btn {
+        flex: 1;
+        margin: 4px;
+        margin-top: 20px;
+
+        button {
```
Comment (byliuyang) [inline frontend/src/component/pages/Home.scss]: I wonder whether we can add  media query to `https://github.com/short-d/short/blob/master/frontend/src/component/ui/Button.scss` to style the Button.

File: frontend/src/component/pages/Home.scss (hunk 4 above)
Comment (rohithbalaji123) [reply frontend/src/component/pages/Home.scss]: Also, can we reduce the placeholder font size so as to accommodate the whole string in the input area?

Comment (kapillamba4) [reply frontend/src/component/pages/Home.scss]: Nice idea @rohithbalaji123, i have reduced the font size from 20px to 16px. It fits completely.

**Comment by kapillamba4** [comment]
@byliuyang @rohithbalaji123 , Why is there a space between circular brackets and Optional in placeholder string "Custom Short Link ( Optional )". It looks odd to me

**Pull request merged**